Development practices/Emails.md

```diff
@@ -0,0 +1,65 @@
+# Guidelines
+
+## Templates
+In most cases we don't need a custom design for the emails. We use [simple HTML templates](https://github.com/mailgun/transactional-email-templates) and modify them slightly with project's theme colors, and add a logo if needed.
+
+If we will need a fully custom template, consult with someone from the JS team on which libraries to use.
+
+If somebody from the other team will be working on emails, please make sure you first set up the Letter opener, prepare the previews for all mailers and instruct your colleague on how to use the [mailer previews](#mailer-previews).
```

Comment:
```suggestion
If somebody from the other team works on emails, please make sure you first set up the Letter opener, prepare the previews for all mailers and instruct your colleague on how to use the [mailer previews](#mailer-previews).
```

Present simple feels better to me here.. @cukoreva what do you think?

Reply:
I definitely agree with not using a future tense, but maybe this would sound better:
```suggestion
If somebody from the other team is supposed to work on emails, please make sure you first set up the Letter opener, prepare the previews for all mailers and instruct your colleague on how to use the [mailer previews](#mailer-previews).
```
or
```suggestion
If somebody from the other team is expected to work on emails, please make sure you first set up the Letter opener, prepare the previews for all mailers and instruct your colleague on how to use the [mailer previews](#mailer-previews).
```